src/WebTestAssertionsTrait.php

```diff
@@ -275,9 +274,9 @@ private static function findElement(string $locator): WebDriverElement
      * @param array $options An array of options to pass to the createKernel method
      * @param array $server  An array of server parameters
      *
-     * @return AbstractBrowser A browser instance
+     * @return KernelBrowser A browser instance
      */
-    protected static function createClient(array $options = [], array $server = []): AbstractBrowser
+    protected static function createClient(array $options = [], array $server = []): KernelBrowser
```

Comment:
This one I am not competely sure if it is correct on Symfony side, but without changing this I got errors too. @dunglas do you know if this change is correct?

Reply:
as far as I can tell, `KernelBrowser` extends `HttpKernelBrowser` extends `AbstractBrowser`, and this has existed since at least 4.4. So changing this to `KernelBrowser` is just "correct: (`WebTestCase::createClient()` has always had this as its `@return`) and it's just more specific: we're narrowing the return type.